helm/prometheus-meta-operator/templates/prometheus-rules/deployment.management-cluster.rules.yml

```
@@ -68,7 +68,7 @@ spec:
       annotations:
         description: '{{`Deployment {{ $labels.namespace}}/{{ $labels.deployment }} is not satisfied.`}}'
         opsrecipe: deployment-not-satisfied/
-      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"aws-operator-.+|cluster-operator-.+|coredns-.+|nginx-ingress-controller-.+|worker-.+|master-.+|app-operator-.+|chart-operator-.+|.+-admission-controller-unique", cluster_id!~"argali|giraffe"} > 0
+      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"aws-operator-.+|azure-operator-.+|cluster-operator-.+|coredns-.+|nginx-ingress-controller-.+|worker-.+|master-.+|app-operator-.+|chart-operator-.+|.+-admission-controller-unique|alertmanager.*|prometheus.*|prometheus-meta-operator.*|promxy.*", cluster_id!~"argali|giraffe"} > 0
```

`prometheus.*` should be enough to catch any prometheus, prometheus-operator and prometheus-meta-operator.
```suggestion
      expr: kube_deployment_status_replicas_unavailable{cluster_type="management_cluster", deployment!~"aws-operator-.+|azure-operator-.+|cluster-operator-.+|coredns-.+|nginx-ingress-controller-.+|worker-.+|master-.+|app-operator-.+|chart-operator-.+|.+-admission-controller-unique|alertmanager.*|prometheus.*|promxy.*", cluster_id!~"argali|giraffe"} > 0
```